shotover-proxy/src/frame/cassandra.rs

```diff
@@ -36,8 +36,8 @@ fn get_batch_len(bytes: &[u8]) -> Result<NonZeroU32> {
     }
 
     let short_bytes = &bytes[1..3];
-    let short = u16::from_be_bytes(short_bytes.try_into()?);
-    Ok(NonZeroU32::new(short.into()).unwrap())
+    let short = u16::from_be_bytes(short_bytes.try_into()?); // it is valid for a batch statement to have 0 statements, but for the purposes of shotover throttling we can count it as one query
```

Comment:
Can we put this comment on a newline?
At the moment it looks like it applies line to line 39 but it actually applies to line 40